### PR TITLE
fix(select): RTL fix for select

### PR DESCRIPTION
**Pull request by sijav**
#### Short description of what this resolves:
RTL fix for select component

#### Changes proposed in this pull request:
- fix padding and margin for RTL on select component

**Ionic Version**: 2.x / 3.x

**Fixes**: #11351

Also there was `.item-wp.item-select ion-label` for windows platform and `.select-md .item-select ion-label` for android which I couldn't reproduce it, if anyone have any idea what is these rules for please let me know.
Thanks.

### CODE REVIEW

File: src/components/select/select.ios.scss [hunk 1]
```diff
@@ -54,3 +58,8 @@ $select-ios-placeholder-color:   $select-ios-icon-color !default;
 
   pointer-events: none;
 }
+
+[dir="rtl"] .select-ios .select-icon .select-icon-inner {
+  right: 5px;
+  left: auto;
+}
```
Comment (AmitMY) [inline src/components/select/select.ios.scss]: the lint rules for ionic indicate a need for a new line at the end of the file.
You can run it locally, `gulp lint.sass`, but you need to install `ruby` for that

**Comment by AmitMY** [comment]
@sijav This looks good.
I will work on border separately, same as all of the other mixins, so there won't be a need to change it per selector.

Please don't close this one though, it is good to know what is missing and where :)

**Comment by sijav** [comment]
@AmitMY Yeah that was the purpose actually :D
Thanks a bunch.

**Comment by AmitMY** [comment]
@brandyscarney No need - will be fixed in a global PR for border property, after the transform one

**Pull request closed without merging**